### PR TITLE
Add `cupy.row_stack` API.

### PR DESCRIPTION
**Pull request by tom24d**
As per title.
Ref: https://github.com/cupy/cupy/issues/6078

### CODE REVIEW

**Comment by tom24d** [comment]
FYI, in numpy, `row_stack` is just an alias of `vstack` https://github.com/numpy/numpy/blob/813a0c11186ded0b5caeb853fd2b22fb9addd511/numpy/lib/shape_base.py#L605

File: cupy/__init__.py [hunk 1]
```diff
@@ -262,6 +262,7 @@ def is_available():
 from cupy._manipulation.join import hstack  # NOQA
 from cupy._manipulation.join import stack  # NOQA
 from cupy._manipulation.join import vstack  # NOQA
+from cupy._manipulation.join import row_stack  # NOQA
```
Comment (emcastillo) [inline cupy/__init__.py:265]:
```suggestion
from cupy._manipulation.join import vstack as row_stack  # NOQA
```

This seems easier! (thanks @asi1024 )

**Review by emcastillo — APPROVED** [review]
/test mini

**Comment by kmaehashi** [comment]
/test mini

(@emcastillo Tests cannot be triggered via review comments 😅 )

**Pull request merged**